src/pages/components/tabs/style.mdx

```diff
@@ -359,6 +363,65 @@ tabs.](images/tab-style-12.png)
   contained tabs.
 </Caption>
 
+### Vertical tab color
+
+| Type       | Element             | Property                                 | Color token              |
+| ---------- | ------------------- | ---------------------------------------- | ------------------------ |
+| Unselected | Tab                 | background-color                         | `$layer` \*              |
+|            |                     | border-bottom, border-right, border-left | `$border-subtle` \*      |
+|            | Label               | text-color                               | `$text-secondary`        |
+|            | Extended background | background-color                         | `$layer` \*              |
+|            |                     | border-right                             | `$border-subtle` \*      |
+| Selected   | Tab                 | background-color                         | `$layer` \*              |
+|            |                     | border-bottom                            | `$border-subtle` \*      |
+|            |                     | border-left                              | `$border-interactive` \* |
```

Comment:
double checking but I don't think `border-subtle` is contextual so it doesn't have `\ *`

Reply:
@thyhmdo i think border styles do have layer tokens looking at components like [dropdown](https://carbondesignsystem.com/components/dropdown/style#color) or [progress indicator](https://carbondesignsystem.com/components/progress-indicator/style#color)